### ISSUE TITLE
Staging Review finding: Unable to access interactive elements with keyboard

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#117014]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Dependent Experience
**Product:** View Dependents
**Feature:** Dependent Verification Tool

## Findings details
**VA.gov Experience Standard - issue:** [User can complete necessary flows.](https://design.va.gov/about/experience-standards/#user-can-complete-necessary-flows)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @rsmithadhoc (Accessibility)

### Description

When tabbing through pages, the focus skips over elements, so keyboard users are unable to interact with those.

<details><summary>Link, screenshot or steps to recreate</summary>

This happens throughout the form, but the gif below shows the intro page.

https://staging.va.gov/view-change-dependents/view

![Image](https://github.com/user-attachments/assets/3390307b-11c2-484a-b115-5ada169e8273)

</details>

## Recommended action

I'm not sure what the cause of this is, it skips links in the footer as well. This will need to be investigated.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** [WCAG 2.1.1 Keyboard - Level A](https://www.w3.org/TR/WCAG22/#keyboard)
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by rsmithadhoc** [comment]
@jasonday Reminded me of this, it actually had to do with a browser setting. I recently switched to a different browser and this setting was not enabled. Once the setting is enabled, tabbing works as expected.

<img width="1776" height="49" alt="Image" src="https://github.com/user-attachments/assets/ac8ed35c-c4ac-42fb-8715-4aac81832458" />

So I will go ahead and close this one out. CC: @shiragoodman for your awareness.
